.gitlab/README.md

```diff
@@ -22,6 +22,7 @@ chmod 700 <scratch_space>/gitlab-runner
 * LLNL: `/usr/workspace/${USER}`
 * OLCF: `$MEMBERWORK/csc449`
 * ALCF: TBD...
+* NERSC: perlmutter: `/pscratch/sd/[FirstLetterOf${USER}]/${USER}`
```

Comment:
let's keep it as a general -> thus remove `perlmutter:`

Reply:
the 'p' in 'pscratch' makes me think it might be perlmutter specific?
https://portal.nersc.gov/cfs/mpccc/sleak/userdocs4/filesystems/perlmutter-scratch/

Reply:
ah, right! then let's keep it as it is